api/db/changelog/db.changelog-45-concept-set.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="danrodney" id="changelog-45">
+
+    <createTable tableName="concept_set">
+      <column name="concept_set_id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="workspace_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="name" type="varchar(80)">
```

Comment:
I realize this would be a departure from other tables, but I'm curious about the decision here to use an internal ID vs using a composite primary key of workspace ID + name

Reply:
Name is mutable, so it can't be the primary key on a concept set.

Reply:
(Also, the query for joining concept_set to concept_set_concept_id is going to be more efficient when using a numeric ID.)